### PR TITLE
Teacher application export cron

### PR DESCRIPTION
**Pull request by bencodeorg**
This PR exports a subset of information about teacher applications to a tab-separated file in a S3 bucket every two hours. The PLC team has a meta application tracker that uses this data to keep track of PD applications received and is currently manually exporting CSVs via Code Studio, which hopefully will be obsolete after this.

Asking @mehalshah for a review as this is similar to `cron/calculate_workshop_survey_results` -- a couple immediate questions: 
1. should main include `if only_one_running?(__FILE__)`?
2. does the active record query raise any red flags from a performance perspective?

### CODE REVIEW

File: bin/cron/teacher_applications_to_s3.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+require_relative '../../dashboard/config/environment'
+require 'cdo/aws/s3'
+require 'csv'
+
+def main
+  applications = Pd::Application::Teacher1819Application.left_outer_joins(:regional_partner).pluck(:course, :name, :locked_at, :status)
```
Comment (sureshc) [inline bin/cron/teacher_applications_to_s3.rb]: I think the Style guideline recommends breaking up lines longer than 80 characters like so:

``` ruby
applications = Pd::Application::Teacher1819Application
  .left_outer_joins(:regional_partner)
  .pluck(:course, :name, :locked_at, :status)
```

Comment (bencodeorg) [reply bin/cron/teacher_applications_to_s3.rb]: Done, thanks!

**Comment by mehalshah** [comment]
I'd suggest including the only-one-running script. It's just good practice.
No concerns on runtime.

Why just Teacher Applications? Do we not need this for facilitator applications?

**Comment by bencodeorg** [comment]
Thanks @mehalshah -- added `only_one_running` check. I checked with Jordyn and she said the team isn't monitoring facilitator applications any more with the same frequency, so teacher applications are the focus.

**Pull request merged**